src/ui/molecules/stripe-elements.svelte

```diff
@@ -194,120 +176,98 @@
-          unsafeElements,
-          brandingInfo?.app_name,
+  const initStripe = async () => {
+    if (stripe) return;
+    if (elements) return;
+
+    try {
+      const { stripe: stripeInstance, elements: elementsInstance } =
+        await StripeService.initializeStripe(
+          gatewayParams,
+          brandingInfo,
+          stripeLocale,
+          stripeVariables,
+          viewport,
         );
+      stripe = stripeInstance;
+      elements = elementsInstance;
 
-        paymentElement.mount("#payment-element");
-
-        paymentElement.on("ready", () => {
-          onLoadingComplete();
-        });
-
-        paymentElement.on(
-          "change",
-          async (event: StripePaymentElementChangeEvent) => {
-            if (
-              taxCollectionEnabled &&
-              event.complete &&
-              event.value.type === "card"
-            ) {
-              await triggerTaxDetailsUpdated();
-            }
-
-            onPaymentInfoChange({
-              complete: event.complete,
-              paymentMethod: event.complete ? event.value.type : undefined,
-            });
-          },
+      const elementsConfiguration = gatewayParams.elements_configuration;
+      if (elementsConfiguration) {
+        await StripeService.updateElementsConfiguration(
+          elements,
+          elementsConfiguration,
         );
-        paymentElement.on("loaderror", (event) => {
-          isMounted = false;
-          onError({
-            code: PaymentElementErrorCode.ErrorLoadingStripe,
-            gatewayErrorCode: event.error.code,
-            message: event.error.message,
-          });
-          onLoadingComplete();
-        });
-      } catch (error) {
-        if (!isMounted) return;
-
-        onError({
-          code: PaymentElementErrorCode.ErrorLoadingStripe,
-          gatewayErrorCode: undefined,
-          message: error instanceof Error ? error.message : String(error),
-        });
-        onLoadingComplete();
       }
-    })();
+    } catch (error) {
+      onStripeElementsLoadingError(error);
+    }
+  };
 
-    return () => {
-      if (isMounted) {
-        isMounted = false;
-        paymentElement?.destroy();
-      }
-    };
+  onMount(async () => {
+    updateStripeVariables();
+    window.addEventListener("resize", onResize);
+    await initStripe();
   });
+
+  onDestroy(() => {
+    window.removeEventListener("resize", onResize);
+  });
+
+  const onPaymentElementChange = async (
+    event: StripePaymentElementChangeEvent,
+  ) => {
+    if (taxCollectionEnabled && event.complete && event.value.type === "card") {
+      await triggerTaxDetailsUpdated();
+    }
+    onPaymentInfoChange({
+      complete: event.complete,
+      paymentMethod: event.complete ? event.value.type : undefined,
+    });
+  };
 </script>
 
-<div id="payment-element"></div>
+<div
+  style="display: flex; flex-direction: column; align-items: stretch; gap: {spacing
+    .gapXLarge[viewport]};"
+>
```

Comment:
Since we are not adding new components in this PR, this should not be needed for now, right?

Reply:
I will be adding the Address and the Email in the follow up ones, but you are right, is not strictly needed in this PR, removing it.

Reply:
Removed.